website/addons/wiki/templates/edit.mako

```diff
@@ -111,7 +111,7 @@
               </div>
           </div>
 
-          % if user['can_edit']:
+          % if has_permission_or_publicly_editable:
```

Comment:
@sloria I get what you're  saying on this one, but this permission does not belong to the user necessarily. It is mostly based on if there is a wiki_settings object variable is_publicly_editable. This seems like unnecessary information for the node to pass every time. Especially since I would have to check if the node has a wiki in order to pass this information. Would it be a viable solution to check "user['can_edit'] or wiki_publicly_editable" separately instead?